models/junos/prefix_lists/prefix_lists.yaml

```diff
@@ -0,0 +1,76 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+  {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+  }
+NETWORK_OS: junos
+RESOURCE: prefix_lists
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: junos_prefix_lists
+  version_added: 2.1.0
+  short_description: Manage prefix-lists attributes of interfaces on Junos devices.
+  description: Manage prefix-lists attributes of interfaces on Junos network devices.
+  author: Rohit Thakur (@rohitthakur2590)
+  requirements:
+    - ncclient (>=v0.6.4)
+    - xmltodict (>=0.12.0)
+  notes:
+    - This module requires the netconf system service be enabled on the device being managed.
+    - This module works with connection C(netconf). See L(the Junos OS Platform Options,../network/user_guide/platform_junos.html).
```

Comment:
This link type no longer works. You need the full URL, not the relative ../network/xxx type.  See Learn  the swiftype search engine used on docs.ansible.com and apply that learning to improving the overall site search capabilities.